dataops/dev_cicd_template.yml

```diff
@@ -2,13 +2,45 @@ variables:
   GITLAB_CICD_IMAGE_REPO: gcr.io/getindata-images-public/cicd-dataops
   GITLAB_CICD_IMAGE_TAG: gcp-0.1.1
```

Comment:
I would suggest bump to the latest tag: `gcp-0.1.4`

https://console.cloud.google.com/gcr/images/getindata-images-public/GLOBAL/cicd-dataops